index.css

```diff
@@ -1,53 +1,63 @@
+*{
+    margin: 0;
+    padding: 0;
+}
 body{
     background-color: aliceblue;
    display: flex;
    justify-content: center;
 }
 .main-container{
-    background-color: rgb(236, 230, 230);
-    height: 100%;
-    width: 20%;
+    background-color: white;
+    width: 30%;
     align-items: center;
     border-radius: 20px;
-    margin-top: 90px;
+    margin-top: 60px;
     display: flex;
     flex-wrap: wrap;
     flex-direction: column;
 }
-.main-container>*{
-    flex-grow: 0;
-    flex-shrink: 1;
-}
+
 img{
-    height: 300px;
-    width: 240px;
-    border-radius: 20px;
-    padding: 10px 10px;
+   width: 85%;
+    border-radius: 2.5em;
+    padding:1.5em;
+  
 }
 h3{
     color: rgb(31, 61, 88);
-    font-size: 25px;
+    font-size: 1.5em;
     font-weight: bold;
-    padding: 0px 5px;
+    padding: 0px 2em;
 }
 p{
     color: rgb(143, 154, 163);
-    padding: 5px;
+    padding: 15px 2.5em;
+    font-size: 20px;
 }
-@media screen and (max-width:300px){
+
+@media screen and (max-width:600px){
```

Comment:
Kindly implement a mobile-first approach for the best web developer practices. create a UI that fits perfectly on mobile then adjust it to fit larger screens using media queries. Am sure this will help you achieve optimal Responsive design